### PR TITLE
Tests: Bluetooth: tester: add BTP command for connecting Proxy Client

### PR DESCRIPTION
**Pull request by KKopyscinski**
This allows to execute MESH/CL/MPXS tests.

### CODE REVIEW

**Review by omkar3141 — APPROVED** [review]

**Review by sjanc — APPROVED** [review]

**Comment by KKopyscinski** [comment]
@omkar3141 your approve is greyed-out, I'm not sure why. After second approve this can be merged

**Review by omkar3141 — APPROVED** [review]

**Comment by omkar3141** [comment]
> @omkar3141 your approve is greyed-out, I'm not sure why. After second approve this can be merged

May be it needs approval from codeowner?

**Comment by KKopyscinski** [comment]
> > @omkar3141 your approve is greyed-out, I'm not sure why. After second approve this can be merged
> 
> May be it needs approval from codeowner?

Maybe. @sjanc can you approve the approval?

**Review by PavelVPV — APPROVED** [review]

**Pull request merged**
